pyproject.toml

```diff
@@ -0,0 +1,66 @@
+[project]
+name = "A2rchi"
+version = "0.1.0"
+description = "An AI Augmented Research Chat Intelligence for MIT's subMIT project in the physics department"
+requires-python = ">=3.7"
+# sorted alphabetically by last name
+authors = [
```

Comment:
Not sure how all this authorship works, do we need to add Tim?

Reply:
Sure thing I'll add Tim -- but to answer your general question I don't think there's a hard rule about this. I was going based off of the set of people who significantly touched the codebase, but your right that we should include my advisor 😂.